src/box/lua/net_box.c

```diff
@@ -82,7 +82,7 @@ enum {
 	/**
 	 * IPROTO protocol version supported by the netbox connector.
 	 */
-	NETBOX_IPROTO_VERSION = 8,
+	NETBOX_IPROTO_VERSION = 9,
```

Comment:
AFAIU net.box does support the feature so you should set the bit in `NETBOX_IPROTO_FEATURES`.

Reply:
Good catch, fixed